article/app/liveblog/LiveBlogPageModel.scala

```diff
@@ -29,7 +40,17 @@ object LiveBlogPageModel {
 
 case class BlockInfo[B](page: Seq[B], self: PageReference)
 
-case class LiveBlogPageModel[+B](blocks: Seq[B], main: Seq[B]/*for key events - TODO remove*/, later: PageReference, earlier: PageReference, canonical: PageReference)
+case class LiveBlogPageModel[+B](
+  blocks: Seq[B],
+  main: Seq[B] /*for key events - TODO remove*/ ,
+  newer: PageReference,
+  older: PageReference,
+  canonical: PageReference,
+  pageNumber: Int,
+  pagesLength: Int,
+  newest: PageReference,
+  oldest: PageReference
+)
```

Comment:
I wasn't sure whether to add these properties to this case class. Let's chat in person when you look at this? :-)